website/server/controllers/api-v3/tasks.js

```diff
@@ -578,11 +765,15 @@ api.scoreCheckListItem = {
  * @apiParam {String} taskId The task _id or alias
  * @apiParam {UUID} itemId The checklist item _id
  *
+ * @apiParam (body) {string} text The text that will replace the current checkitem text.
+ *
+ * @apiParamExample {json} Example body:
+ * {"text":"Czech 1"}
+ *
  * @apiSuccess {Object} data The updated task
  *
  * @apiUse TaskNotFound
  * @apiUse ChecklistNotFound
- * @apiUse ChallengeNotFound
```

Comment:
I see the code, but I can't produce the error. In order to throw the error, challengeId must have data in it, but turn out to have an invalid Id... I've tried creating a challenge, creating the task, then deleting the challenge, but it still passes. Under what circumstances would there be a challengeId in a task that does not return a valid challenge?